populatedb.js

```diff
@@ -1,229 +1,207 @@
 #! /usr/bin/env node
 
-console.log('This script populates some test books, authors, genres and bookinstances to your database. Specified database as argument - e.g.: node populatedb "mongodb+srv://cooluser:coolpassword@cluster0.lz91hw2.mongodb.net/local_library?retryWrites=true&w=majority"');
+console.log(
+  'This script populates some test books, authors, genres and bookinstances to your database. Specified database as argument - e.g.: node populatedb "mongodb+srv://cooluser:coolpassword@cluster0.lz91hw2.mongodb.net/local_library?retryWrites=true&w=majority"'
```

Comment:
@bsmth FYI This particular instance is intentional and "good". Why? This script is not educational code. It is provided very early in the process as dumb "download and run me" script that you use just once to populate the DB so that users can see what the views look like before they have got to the forms part where they know how to properly create entries. 
The URL that is uses is provided by mongoDB. 

Of course if this was educational or intended to ever be part of the example or description, you'd be right.